repository/flink/docs/demo/financial-fraud/README.md

```diff
@@ -19,100 +19,98 @@ The architecture follows more or less the [SMACK stack architecture](https://mes
         ```
-        *Note: Don't forget to say "**no**" when prompted, we just want to install the Framework and FrameworkVersion objects but we don't want to instantiate an Instance object*
-- Have the `kafka` Framework with `0.1.0` as FrameworkVersion installed 
+- Have the `kafka` Operator with `0.1.1` as OperatorVersion installed 
     - Use the KUDO CLI with the following command:
         ```bash
-        $ kubectl kudo install kafka --package-version=0.1.0
-        framework.kudo.k8s.io/v1alpha1/kafka created
-        frameworkversion.kudo.k8s.io/v1alpha1/kafka-0.1.0 created
-        No Instance tied to this "kafka" version has been found. Do you want to create one? (Yes/no) no
-
+        $ kubectl kudo install kafka --version=0.1.1 --skip-instance
+        operator.kudo.k8s.io/v1alpha1/kafka created
+        operatorversion.kudo.k8s.io/v1alpha1/kafka-0.1.1 created
+        ```
+- Have the `flink` Operator with `0.1.0` as OperatorVersion installed 
+    - Use the KUDO CLI with the following command:
+        ```bash
+        $ kubectl kudo install flink --version=0.1.0 --skip-instance
+        operator.kudo.k8s.io/v1alpha1/flink created
+        operatorversion.kudo.k8s.io/v1alpha1/flink-0.1.0 created
         ```
-        *Note: Don't forget to say "**no**" when prompted, we just want to install the Framework and FrameworkVersion objects but we don't want to instantiate an Instance object*
-- Have the `flink` Framework with `0.2.0` as FrameworkVersion installed 
-    - Get the KUDO Frameworks repository: `git clone git@github.com:kudobuilder/frameworks.git`
-    - Change directory into the cloned repository: `cd kudo`
-    - Install the Flink `0.2.0` objects straight out the repository:
-        - `kubectl create -f repository/flink/0.2.0/flink-framework.yaml`
-        - `kubectl create -f repository/flink/0.2.0/flink-frameworkversion.yaml`
         
-Now you should have all required Frameworks installed.
+Now you should have all required Operators installed.
 
 ## Getting Started
 
-Install the `flink-financial-demo` from the main repository directory via:
+Install the Flink `financial-fraud` demo from the main repository directory.
 
-`kubectl create -f repository/flink/docs/demo/financial-fraud/flink-demo.yaml`
+ - Get the KUDO Operators repository: `git clone git@github.com:kudobuilder/operators.git`
+    - Change directory into the cloned repository: `cd operators`
+    - Install the Flink demo objects straight out of the repository:
+        ```bash
+        $ kubectl kudo install repository/flink/docs/demo/financial-fraud/demo-operator --instance flink-demo
+        operator.kudo.k8s.io/v1alpha1/flink-demo created
+        operatorversion.kudo.k8s.io/v1alpha1/flink-demo-0.1.0 created
+        No instance named 'flink-demo' tied to this 'flink-demo' version has been found. Do you want to create one? (Yes/no) yes
+        instance.kudo.k8s.io/v1alpha1/flink-demo created
+        ```
```

Comment:
A few commands that the user could run at this point to demonstrate progress would be really cool here, i.e `kubectl kudo plan status --instance flink-demo-zk`.

Reply:
> A few commands that the user could run at this point to demonstrate progress would be really cool here, i.e `kubectl kudo plan status --instance flink-demo-zk`.

Perfect, I added it here: https://github.com/kudobuilder/operators/pull/48/commits/82e2eebdd494564f84141e11f81310e70017c6fa